frame/beefy/src/lib.rs

```diff
@@ -99,12 +100,15 @@ pub mod pallet {
 impl<T: Config> Pallet<T> {
 	/// Return the current active BEEFY validator set.
 	pub fn validator_set() -> Option<ValidatorSet<T::BeefyId>> {
-		let validators: Vec<T::BeefyId> = Self::authorities();
+		let validators: WeakBoundedVec<T::BeefyId, T::MaxAuthorities> = Self::authorities();
```

Comment:
There should be no need for weak bounded vec anymore. Now we can always return an error if the size is not right